ckanext/datawagovautheme/templates/snippets/facet_list.html

```diff
@@ -6,3 +6,38 @@ <h2 class="module-heading">
     {{ title }}
   </h2>
 {% endblock %}
+          {% block facet_list_items %}
```

Comment:
What do these additions do?

Reply:
this bit is overriding code to fix ellipsis issue for sidebar on dataset page. in your case it will be https://uat.landgate.links.com.au/dataset left sidebar.

Reply:
Ah, I see now. So this is code from CKAN core's `facet_list.html` that we're dropping in here to remove truncation?